### ISSUE TITLE
The training code

### ISSUE THREAD

**Issue by bolimath** [uclaopt/Provable_Plug_and_Play#1]
Can you provide the original  training code? Which loss function is used to train the network? Thanks!

**Comment by wotaoyin** [comment]
Hi,

The pretrained models and PnP codes were uploaded before ICML.
The training part is done by co-authors Sicheng and Xiaohan. They are on
vacation. I will ask them to upload soon.

Wotao

**Comment by wotaoyin** [comment]
The training codes have been uploaded to the repository.

Cheers,
Wotao

**Comment by bolimath** [comment]
Thanks a lot.
